_data/events/Manila.json

```diff
@@ -0,0 +1,18 @@
+{
+  "title": "Nokia Manila TC Coderetreat 2017",
+  "url": "https://www.yammer.com/nokia.com/#/threads/inGroup?type=in_group&feedId=8757214&view=all",
```

Comment:
This URL is redirecting to a microsoftonline.com login screen. The URL should be public so that people can easily register.